src/features/book/book.module.ts

```diff
@@ -1,11 +1,12 @@
 import { Module } from '@nestjs/common';
 import { BookService } from './book.service';
 import { BookController } from './book.controller';
-import { UserModule } from '../user/user.module';
+//import { UserModule } from '../user/user.module';
+import { BookJsonRepository } from './book.json-repository';
 
 @Module({
   controllers: [BookController],
-  providers: [BookService],
-  imports: [UserModule],
+  providers: [BookService, BookJsonRepository],
+  exports: [BookService],
```

Comment:
Do we need use bookService in different module ?

Reply:
fixed

Reply:
What are you fixing here?

Reply:
add bookJsonRepository in providers and export Bookservice.
deleted unused imports.

Reply:
Do we really need use BookService outside the Book Module ?